tests/checks/cell/test_ascii_value.py

```diff
@@ -0,0 +1,31 @@
+from frictionless import validate, checks
+
+
+# Issues
+
+
+def test_validate_ascii_value_845():
```

Comment:
BTW don't forget that you need to skip these tests for Python3.6 (see other skipped tests for reference)

Reply:
sure!. Sorry I forgot to add it in last commit.